.github/workflows/mac.yml

```diff
@@ -0,0 +1,21 @@
+name: macOS build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build on macOS
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Setup Environment
+      run: |
+        scripts/mac-toolchain-build /usr/local
+        
+    - name: Build HandBrake
+      run: |
+        ./configure --launch-jobs=$(sysctl -n hw.ncpu) --launch
```

Comment:
We should configure with `--harden` going forward. I believe it is required for Apple notarization.

Reply:
The --harden option is unrelated to the "hardened runtime" required by notarization.

Reply:
Ah, thanks for clarifying.